app/.env.sample

```diff
@@ -1,2 +1,10 @@
-ENV_TYPE=
-GOOGLE_API_KEY=
\ No newline at end of file
+ENV_TYPE=''
+PROJECT_ID=''
+
+CLIENT_ID = '' #Reddit credentials
```

Comment:
Change the name of the env. variables adding prefixes like `REDDIT_` for better readability of this.